packages/dso-toolkit/package.json

```diff
@@ -56,16 +56,16 @@
     "gulp-svgstore": "^9.0.0",
     "gulp-typescript": "^6.0.0-alpha.1",
     "list-stream": "^2.1.0",
-    "markdown-to-jsx": "^7.5.0",
+    "markdown-to-jsx": "^7.6.2",
     "plugin-error": "^2.0.1",
-    "postcss": "^8.4.47",
+    "postcss": "^8.4.49",
     "prettier": "^3.3.3",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "sass": "1.80.6",
+    "sass": "1.81.0",
```

Comment:
Werkt de color:channel functie nu misschien wel? Dit is het issue waar @hansgrimm momenteel mee bezig is #2846

Reply:
nee helaas niet

Reply:
Da's jammer. Misschien komt Hans ergens mee.